### PR TITLE
Design files

### PR DESCRIPTION
**Pull request by rafaelmedian**
### Things added
1. Hack font
2. webpack.svg
3. sketch files
-  Simple Branding
- Terminal and Editor states
-  Website

### CODE REVIEW

**Comment by rmariuzzo** [comment]
Hey @zxhadow! Thanks for your hard work! I love the design in Sketch for the dark editor and the CLI. However, I have some remarks about the usage of this plugin.
1. The user install the plugin locally doing:
   
   **`shell`**
   
   ``` sh
   npm install config-webpack-plugin --save-dev
   ```
2. The user configure the plugin in his/her `webpack.config.js` telling to the plugin which config file to intercept during webpack's compilation time.
   
   **`webpack.config.js`**
   
   ``` js
   const ConfigPlugin = require('webpack-config-plugin');
   
   module.exports = {
       // Among other stuff...
       plugins: [ new ConfigPlugin('./config.js') ]
   }
   ```
3. The user trigger webpack's compilation running: `webpack` command. **During that compilation time, the plugin will read the specified configuration file and will merge matching properties with current environment variables and will ‘write the changes to disk’**.

That said, I think we need to help the audience to understand what the plugin does in a simple way that is: intercept a config file, replace config properties with matching variable and, write the modified config file disk.

---

**TL;DR.**
- I ❤️ the banner style for the branding. The logo of webpack along with the blank hexagon makes a good job transmitting the message of a **plugin**.
- The ‘process-env‘ from Windows Pages are not accurate. The user don't need to type `process.env` (that's just a property from inside Node.js). In the ‘how it works’ image that I prepared I was trying to illustrate what are the current environment variables that are visible to NodeJS typing `process.env` inside a NodeJS session. Maybe, I failed in transmitting that idea. Apart, from the installation, the only required command to kickoff the plugin is: `webpack` which can be ignored because that's how all webpack plugin runs.
- From my previous point. We need to show the user how a config file looks with props, some current environment variables and the result of the tiny work of the plugin.

Please, don't hesitate to ask anything about this entire comment (or blog post) 😅

**Comment by rafaelmedian** [comment]
Thanks @rmariuzzo for this well explained comment. See these states for clarification if I got it right this time.

These are not the explanation itself, but we will use them to create a little gif or another better idea to do it. 

screen shot 2016-08-12 at 7 31 49 pm

**Comment by rmariuzzo** [comment]
Ok, I like the idea for the GIF and the graphics.

**Comment by rafaelmedian** [comment]
I will be very busy this week, but we can close this PR and create another one with the assets so we could upload the files in case someone wants to help with them.

**Comment by rmariuzzo** [comment]
@zxhadow I will create the `gh-pages` branch which will hold the landing page and the sources files. That way we don't mix them with the library.
